### PR TITLE
feat: new independent showcase page

### PR DESCRIPTION
**Pull request by bzp2010**
Changes:

Move the user wall on the home page to a separate page to meet ASF's regulations for the project website.

Screenshots of the change:

image

### CODE REVIEW

**Comment by juzhiyuan** [comment]
cc @Baoyuantop @LiteSun to have a review

File: website/src/pages/showcase/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          Use cases
+        </div>
+        <div className="tips">
+          This project is used by all these folks
+          <br />
+          Are you using this project?&nbsp;
+          <a
+              href="https://github.com/apache/apisix/blob/master/powered-by.md"
+              target="_blank"
+              rel="noopener"
```
Comment (liuxiran) [inline website/src/pages/showcase/index.js:29]:
```suggestion
              rel="noopener external nofollow"
```
would be better to focus on our own flow and not brain it to the outside

**Comment by 1502shivam-singh** [comment]
Hey @bzp2010 this version looks nice and clean, but I feel we can work more to build over this in the future. Personally I was thinking something on the lines of [this page](https://stripe.com/en-in/customers) with the case studies linking to blogs on our websites or their's related to the work.

For now, this version does the work and looks clean too, so I feel we can keep it and moving forward build on it's foundation.

On a side note, given that this page is clear, what you all think about my suggestion in #519 on showcasing our users ?

**Comment by bzp2010** [comment]
Hi, @1502shivam-singh.
I saw your explanation there. I think this is very good. At the same time, we can fold a part of the logo by supporting downward expansion. However, this requires further understanding and interviews with users. This may take some time to move forward gradually.

**Comment by liuxiran** [comment]
Hi @1502shivam-singh , your design is good way IMO, and in apache way, we need a new page to show the user cases in stead of showing them in home page, so It will be very kind of you to give more advices on the single page design, thanks a lot

**Comment by liuxiran** [comment]
sorry to make `nofollow` to the wrong link, just modify them @bzp2010

**Comment by 1502shivam-singh** [comment]
> Hi @1502shivam-singh , your design is good way IMO, and in apache way, we need a new page to show the user cases in stead of showing them in home page, so It will be very kind of you to give more advices on the single page design, thanks a lot

Sure, one small suggestion I have as of now is to redirect the learn more links to their respective case studies, but as mentioned by @bzp2010 above that these are on availability basis, so what we can do is link the ones we have and keep the rest as is. As time progresses we could adapt changes later. 
As of visual design, we need to work on this, but firstly having something up is important so improving visual design can be done later I guess.

**Comment by 1502shivam-singh** [comment]
> Hi, @1502shivam-singh.
> I saw your explanation there. I think this is very good. At the same time, we can fold a part of the logo by supporting downward expansion. However, this requires further understanding and interviews with users. This may take some time to move forward gradually.

Gotcha, we can add data gradually as we have more to leverage.

**Comment by liuxiran** [comment]
> > Hi @1502shivam-singh , your design is good way IMO, and in apache way, we need a new page to show the user cases in stead of showing them in home page, so It will be very kind of you to give more advices on the single page design, thanks a lot
> 
> Sure, one small suggestion I have as of now is to redirect the learn more links to their respective case studies, but as mentioned by @bzp2010 above that these are on availability basis, so what we can do is link the ones we have and keep the rest as is. As time progresses we could adapt changes later.
> As of visual design, we need to work on this, but firstly having something up is important so improving visual design can be done later I guess.

Nice suggestion @1502shivam-singh , we have some of the use case blogs in our website, they can be as the user case studies here, not many though. How about we use this page as the user case show page, and at the same time use it as the user case index page?
also cc @Serendipity96 @yzeng25 @juzhiyuan @Baoyuantop @KishaniKandasamy @moonming

**Comment by liuxiran** [comment]
Hi all, we have finished our first step of update blog page, here is our action:

File: website/src/pages/showcase/styles.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        display: flex;
+        flex-direction: column;
+        align-items:center;
+        box-sizing: border-box;
+        position: relative;
+
+        .logo {
+            width: 100px;
+            max-height: 40px;
+        }
```
Comment (Baoyuantop) [inline website/src/pages/showcase/styles.scss]: I noticed that some square or round logos are deformed. How about `max-width: 100px` ? Or is there a better way？

Comment (bzp2010) [reply website/src/pages/showcase/styles.scss]: Yes, I think so.
fixed

**Comment by bzp2010** [comment]
Hi, @liuxiran. I have modified the showcase page to the style in the connection. [link here](https://github.com/apache/apisix-website/issues/519#issuecomment-913367605), and [demo here](https://deploy-preview-547--apisix.netlify.app/showcase/)

In the process of modification, I found that some logo images are non-standard shapes. (e.g. square or too wide) These pictures don't show well. We need to notify these users to update the logo image. If there is no response, it should be removed or hidden.

**Comment by liuxiran** [comment]
> In the process of modification, I found that some logo images are non-standard shapes. (e.g. square or too wide) These pictures don't show well. We need to notify these users to update the logo image. If there is no response, it should be removed or hidden.

LGTM, we can create a new issue to trace it

and it would be better to keep the name the same as the url, use `showcase` would be better then, or unified into powered by is also a good choice.

**Comment by liuxiran** [comment]
Hi @bzp2010 , please fix the conflict file when you have time , thanks

**Review by liuxiran — APPROVED** [review]

**Comment by liuxiran** [comment]
cc @Baoyuantop @1502shivam-singh to take a look, thanks

**Comment by bzp2010** [comment]
Update: Add a timeout for Axios to prevent the detection process from executing indefinitely.

cc @liuxiran @juzhiyuan

**Review by Baoyuantop — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**